docs/source/schema.rst

```diff
@@ -531,3 +531,22 @@ Available events
       "delay": "1h"
     }
   }
+
+Custom Isolation Level
+-------------------------
+
+You can specify transaction isolation level for api request.
+Currently, this is only supported for mysql.
+By default, level for read is repeatable read and the others are
```

Comment:
The default setting is "read repeatable" for read operations and "serializable" for operations that modify the database (create, update, delete).